src/shared/api/http.ts

```diff
@@ -59,6 +61,11 @@ export function fetchBlocks(params?: PaginationParams): Promise<Paginated<BlockS
   return get('/blocks', params);
 }
 
+export async function fetchBlock(height: number): Promise<BlockShallow> {
+  // FIXME: replace with real api
+  return makeBlock();
+}
```

Comment:
Mocks are used in fake http.

```suggestion
export async function fetchBlock(heightOrHash: number | string): Promise<BlockShallow> {
  return get(`/blocks/${heightOrHash}`)
}
```